### PR TITLE
Enable LUKS2 and PBKDF2 by default for compat with grub (boo#1185291)

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by lnussel** [comment]
is the ci using an outdated yast2-installation-control? Works for me locally

**Comment by ancorgs** [comment]
I'm not saying that will fix the problem with CI, but it would be probably correct to update the version of the BuildRequires for yast2-installation-control.

**Comment by ancorgs** [comment]
> is the ci using an outdated yast2-installation-control? Works for me locally

CI uses a container that, according to registry.opensuse.org, has not been rebuilt since 11. Apr 2024. Causes are under investigation.

**Comment by lnussel** [comment]
right, at least it will make the problem obvious. updated.

**Comment by lnussel** [comment]
haha, it's building a package o_O

**Comment by lslezak** [comment]
I have fixed building the container images in OBS and restarted the CI, now it is green. :white_check_mark:

File: package/skelcd-control-openSUSE.spec [hunk 1]
```diff
@@ -39,7 +39,7 @@ BuildRequires:  libxml2-tools
 # xsltproc
 BuildRequires:  libxslt-tools
 # Added 'lsm' section (jsc#SLE-22069)
```
Comment (ancorgs) [inline package/skelcd-control-openSUSE.spec:41]: This comment refers to the reason for requiring that version of yast2-installation-control. Now it's outdated.

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**